src/rgw/rgw_op.cc

```diff
@@ -4793,16 +4810,41 @@ void RGWPostObj::execute(optional_yield y)
       emplace_attr(RGW_ATTR_COMPRESSION, std::move(tmp));
     }
 
-    /* TODO:  implement POST checksums */
+    if (false && cksum_filter) {
```

Comment:
This will never run?

Reply:
looks like I intended to come back to that--thanks for spotting it